lmp/build.sh

```diff
@@ -52,7 +52,10 @@ export FACTORY_SSTATE_CACHE_MIRROR="/var/cache/bitbake/v${LMP_VERSION_CACHE}-sst
 su builder -c $HERE/bb-config.sh
 touch ${archive}/customize-target.log && chown builder ${archive}/customize-target.log
 touch ${archive}/bitbake_buildchart.svg && chown builder ${archive}/bitbake_buildchart.svg
-touch ${archive}/bitbake_debug.log ${archive}/bitbake_warning.log ${archive}/bitbake_buildstats.log && chown builder ${archive}/bitbake_*.log
+touch ${archive}/bitbake_debug.log \
+	${archive}/bitbake_warning.log \
+	${archive}/bitbake_buildstats.log \
+	&& chown builder ${archive}/bitbake_*.log
```

Comment:
Please separate this change in another pr, it has no relationship to the change proposed by this pr.

Reply:
This should be fine, noticed the related change in a later commit.